e2e/rbd.go

```diff
@@ -821,6 +829,112 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("test RBD volume encryption with user secrets based SecretsMetadataKMS", func() {
+				err := deleteResource(rbdExamplePath + "storageclass.yaml")
+				if err != nil {
+					e2elog.Failf("failed to delete storageclass with error %v", err)
+				}
+				scOpts := map[string]string{
+					"encrypted":       "true",
+					"encryptionKMSID": "user-ns-secrets-metadata-test",
+				}
+				err = createRBDStorageClass(f.ClientSet, f, defaultSCName, nil, scOpts, deletePolicy)
+				if err != nil {
+					e2elog.Failf("failed to create storageclass with error %v", err)
+				}
+
+				// user provided namespace where secret will be created
+				namespace := tenantNamespace
```

Comment:
why to create new variable here?

Reply:
It's to be more precise about which namespace is being used to fetch secret / kms config type.

Reply:
why not use the `tenantNamespace` variable directly in other places instead of creating an alias and using it?

Reply:
@Madhu-1 , I used another variable `nsmespace` at 
 ```go
// user provided namespace where secret will be created
namespace := cephCSINamespace
...&
// PVC creation namespace where secret will be created
namespace := f.UniqueName	
```
to highlight the difference with the two e2e testcases. Does that pass as an explanation for its usage :sweat_smile: ?

Reply:
it's not a major issue but why? the `By` context already explains what you are trying to do.

Reply:
IMO, It may not be clear at first look, why secrets are being created in respective namespaces `cephCSINamespace` & `f.Unique` if  its being directly used.